packages/now-cli/test/fixtures/unit/big-file/package.json

```diff
@@ -1,6 +0,0 @@
-{
-  "name": "",
-  "version": "0.0.1",
-  "description": "",
-  "dependencies": {}
-}
```

Comment:
Why is this `big-file` test deleted? It doesn't look like v1?

Reply:
It's not being used